code/game/objects/structures/bookcase.dm

```diff
@@ -90,7 +113,7 @@
 	update_icon()
 
 /obj/structure/bookcase/manuals/research_and_development
-	name = "R&D Manuals bookcase"
+	name = "R&D manuals bookcase"
```

Comment:
This needs to explicitly be improper because R is capitalized.

Reply:
done